src/test/java/org/openrewrite/quarkus/quarkus2/JavaEEtoQuarkus2MavenDependenciesMigrationTest.java

```diff
@@ -0,0 +1,404 @@
+          //language=xml
+          pomXml(
+            """
+              <?xml version="1.0" encoding="UTF-8"?>
+              <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+                <modelVersion>4.0.0</modelVersion>
+                <groupId>org.openrewrite.sample</groupId>
+                <name>Sample Java EE7 EJB Module</name>
+                <artifactId>ee7-ejb</artifactId>
+                <version>1.0-SNAPSHOT</version>
+                <packaging>war</packaging>
+                            
+                <properties>
+                  <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+                  <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+                  <maven.compiler.source>1.8</maven.compiler.source>
+                  <maven.compiler.target>1.8</maven.compiler.target>
+                </properties>
+                            
+                <dependencies>
+                  <dependency>
+                    <groupId>javax</groupId>
+                    <artifactId>javaee-api</artifactId>
+                    <version>7.0</version>
+                    <scope>provided</scope>
+                  </dependency>
+                  <dependency>
+                    <groupId>javax.annotation</groupId>
+                    <artifactId>javax.annotation-api</artifactId>
+                    <version>1.3.2</version>
+                    <scope>provided</scope>
+                  </dependency>
+                </dependencies>
+              </project>
+              """,
+            """
+              <?xml version="1.0" encoding="UTF-8"?>
+              <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+                <modelVersion>4.0.0</modelVersion>
+                <groupId>org.openrewrite.sample</groupId>
+                <artifactId>ee7-ejb</artifactId>
+                <version>1.0-SNAPSHOT</version>
+                <name>Sample Java EE7 EJB Module</name>
+
+                <properties>
+                  <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+                  <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+                  <maven.compiler.source>11</maven.compiler.source>
+                  <maven.compiler.target>11</maven.compiler.target>
+                </properties>
+                <dependencyManagement>
+                  <dependencies>
+                    <dependency>
+                      <groupId>io.quarkus.platform</groupId>
+                      <artifactId>quarkus-bom</artifactId>
+                      <version>2.16.12.Final</version>
```

Comment:
For that we use `SourceSpec.after(UnaryOperator<String>)`, as seen in [this example for ArchUnit](https://github.com/openrewrite/rewrite-testing-frameworks/blob/ab53509abe5cfc21205004bdcb79e84d42281f40/src/test/java/org/openrewrite/java/testing/archunit/ArchUnit0To1MigrationTest.java#L61-L82). It's not ideal for assertions in large text blocks, so it might make sense to break up the `convertJavaEEToQuarkusDependencies1` test (or even the recipe) into more easily testable parts.